parallel-install/pkg/postuninstaller/postuninstaller.go

```diff
@@ -0,0 +1,161 @@
+package postuninstaller
+
+import (
+	"context"
+	"github.com/avast/retry-go"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/preinstaller"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/client-go/dynamic"
+)
+
+// Config defines configuration values for the PostUninstaller.
+type Config struct {
+	Log                      logger.Interface        //Logger to be used
+	KubeconfigSource         config.KubeconfigSource //KubeconfigSource to be used
+	InstallationResourcePath string
+}
+
+//go:generate mockery --name PostUninstaller
+
+// PostUninstaller removes leftover resources from k8s cluster, added during Kyma installation.
+type PostUninstaller struct {
+	cfg             Config
+	dynamicClient   dynamic.Interface
+	resourceManager preinstaller.ResourceManager
+	retryOptions    []retry.Option
+}
+
+// Output contains lists of Deleted and NotDeleted resources during PostUninstaller uninstallation.
+type Output struct {
```

Comment:
i don't see a reason for having this information tbh

Reply:
This kind of information was provided by `PreInstaller` too, but now when I change the strategy to bulk delete all labeled resources it would make no sense to fetch the ones that remained because of error, so the struct will be removed. I will consider removing it from `PreInstaller` too.